earth_enterprise/src/server/examplesearch_2delete.sql

```diff
@@ -0,0 +1,19 @@
+--
```

Comment:
I don't love the idea of having a duplicate file but I see the need for it and I can't think of anything better. It would be nice to eliminate the duplication, though.